Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1239,7 +1239,7 @@ issueFacedAndRateView state push =
       , height WRAP_CONTENT
       , weight 1.0
       , gravity BOTTOM
-      , visibility if not state.data.ratingViewState.issueFacedView || state.data.ratingViewState.doneButtonVisibility then VISIBLE else GONE
+      , visibility if (not state.data.ratingViewState.issueFacedView || state.data.ratingViewState.doneButtonVisibility || (state.data.rateCard.nightCharges && state.data.ratingViewState.selectedYesNoButton == 0)) && not (state.data.ratingViewState.selectedYesNoButton == 1 && state.data.rateCard.nightCharges) then VISIBLE else GONE
```

Comment:
@mohdshoaib18169 is can we modify one variable for its visibility instead of using selectedYesNoButton as a parameter .